app/models/manageiq/providers/azure_stack/inventory.rb

```diff
@@ -6,6 +6,21 @@ class ManageIQ::Providers::AzureStack::Inventory < ManageIQ::Providers::Inventor
   # Default manager for building collector/parser/persister classes
   # when failed to get class name from refresh target automatically
   def self.default_manager_name
-    'CloudManager'
+    'CloudManager' # TODO; version?
```

Comment:
I'd just use 2018 here

Reply:
EDIT: As suggested below, the base class would contain 2018 code anyway, so we can just leave it at `CloudManager`

Reply:
I was wondering whether this would be okay, given that the manager's `api_version` points to the older profile, while this method would statically return the newer version (not entirely sure if and when this could happen though). But if we extract the behavior of the latest version into the base class and just return `'CloudManager'` here, this makes even more sense to me.